src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -711,6 +711,13 @@ private async Task<(bool Success, string ErrorMessage)> SetTriggersAsync(string
                 request.Headers.Add(ScriptConstants.SiteTokenHeaderName, token);
                 request.Content = new StringContent(content, Encoding.UTF8, "application/json");
 
+                if (_environment.IsManagedAppEnvironment())
+                {
+                    request.Headers.Add("K8SE-APP-NAME", _environment.GetEnvironmentVariable("CONTAINER_APP_NAME"));
+                    request.Headers.Add("K8SE-APP-NAMESPACE", _environment.GetEnvironmentVariable("CONTAINER_APP_NAMESPACE"));
+                    request.Headers.Add("K8SE-APP-REVISION", _environment.GetEnvironmentVariable("CONTAINER_APP_REVISION"));
+                }
+
```

Comment:
I think the following if clause should now be an else?

Reply:
Changed the clause to 'else if'